pangeo_forge_recipes/recipes/xarray_zarr.py

```diff
@@ -14,12 +14,14 @@
 from typing import Callable, Dict, Hashable, Iterator, List, Optional, Sequence, Set, Tuple
 
 import dask
+import fsspec
 import numpy as np
 import xarray as xr
 import zarr
 
 from ..chunk_grid import ChunkGrid
 from ..patterns import CombineOp, DimIndex, FilePattern, Index
+from ..reference import create_hdf5_reference, unstrip_protocol
```

Comment:
Note that `_unstrip_protocol` is now available in `fsspec.utils`. A more thorough one for chained FSs would be good, but not necessary for the work here.